<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Web Socket Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="messageText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>441</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Write a your name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOk">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>83</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="listMessages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
